website/docs/quickstart/security.md

```diff
@@ -0,0 +1,510 @@
+        authorizer.enabled: true
+        super.users: User:admin
+    ports:
+        - "9124:9123"
+    volumes:
+      - shared-tmpfs:/tmp/fluss
+  zookeeper:
+    restart: always
+    image: zookeeper:3.9.2
+
+volumes:
+  shared-tmpfs:
+    driver: local
+    driver_opts:
+      type: "tmpfs"
+      device: "tmpfs"
+```
+
+#### Launch the components
+
+Save the `docker-compose.yaml` script and execute the `docker compose up -d` command in the same directory
+to create the cluster.
+
+Run the below command to check the container status:
+
+```bash
+docker container ls -a
+```
+
+#### Prepare Flink Environment
+##### Start Flink Cluster
+You can start a Flink standalone cluster refer to [Flink Environment Preparation](engine-flink/getting-started.md#preparation-when-using-flink-sql-client)
+
+**Note**: Make sure the [Fluss connector jar](/downloads/) already has copied to the `lib` directory of your Flink home.
+```shell
+bin/start-cluster.sh 
+```
+
+
+##### Enter into SQL-Client
+Use the following command to enter the Flink SQL CLI Container:
+```shell
+bin/sql-client.sh
+```
+
+### Create Catalogs for Each User
+Create separate catalogs for each user:
+```sql title="Flink SQL"
+CREATE CATALOG admin_catalog WITH (
+'type' = 'fluss',
+'bootstrap.servers' = 'localhost:9123',
+'client.security.protocol' = 'SASL',
+'client.security.sasl.mechanism' = 'PLAIN',
+'client.security.sasl.username' = 'admin',
+'client.security.sasl.password' = 'admin-pass'
+);
+```
+
+```sql title="Flink SQL"
+CREATE CATALOG producer_catalog WITH (
+'type' = 'fluss',
+'bootstrap.servers' = 'localhost:9123',
+'client.security.protocol' = 'SASL',
+'client.security.sasl.mechanism' = 'PLAIN',
+'client.security.sasl.username' = 'producer',
+'client.security.sasl.password' = 'producer-pass'
+);
+
+```
+
+```sql title="Flink SQL"
+CREATE CATALOG consumer_catalog WITH (
+'type' = 'fluss',
+'bootstrap.servers' = 'localhost:9123',
+'client.security.protocol' = 'SASL',
+'client.security.sasl.mechanism' = 'PLAIN',
+'client.security.sasl.username' = 'consumer',
+'client.security.sasl.password' = 'consumer-pass'
+);
+```
+
+
+### Add ACLs for Producer and Consumer
+As the `admin` user, add ACLs to grant permissions:
+
+Allow `produce`r to write data:
+```sql
+-- This can used for flink 1.18 and above.
+CALL admin_catalog.sys.add_acl(
+    'cluster', 
+    'ALLOW',
+    'User:producer', 
+    'WRITE',
+    '*'
+);
+
+-- This can only used for flink 1.19 and above.
+CALL admin_catalog.sys.add_acl(
+    resource => 'cluster', 
+    permission => 'ALLOW',
+    principal => 'User:producer', 
+    operation => 'WRITE',
+    host => '*'
+);
+```
+
+Allow `consumer` to read data:
+```sql
+-- This can used for flink 1.18 and above.
+CALL admin_catalog.sys.add_acl(
+    'cluster', 
+    'ALLOW',
+    'User:consumer', 
+    'READ',
+    '*'
+);
```

Comment:
We don't need the legacy syntax if using flink docker images which is 1.20 by defaut.